policyengine_us/parameters/gov/states/va/dss/tanf/grant_standard/group2/addition.yaml

```diff
@@ -0,0 +1,12 @@
+description: VA provides extra assistance for each person above 10 in a family living in group II localities of this amount.
+
+values:
+  2020-07-01: 81
+
+metadata:
+  unit: currency-USD
+  period: month
+  label: VA TANF extra assistance for group II localities
+  reference:
+    - title: VA TANF Guidance Manual - Need and Amount of Assistance 
```

Comment:
Please add the page number of the source for all yaml files.

Reply:
we specify page numbers in urls as below